src/services/HackParserService.ts

```diff
@@ -0,0 +1,31 @@
+import { generateS3PresignedUrl, listKeysWithPrefixWithinS3, uploadFileToS3 } from '@/utils/s3';
+
+/**
+ * Returns whether the HackParser integration is enabled
+ */
+export function HackParserIntegrationEnabled() {
+  return !!process.env.HACK_PARSER_BUCKET_NAME;
+}
+
+/**
+ * Generates a presigned URL for a HackParser S3 object; lasts for 2 days
```

Comment:
🤔 I'm wondering if 2 days is enough? Like I accept on a Friday morning and it's due end of day Monday it may not be viewable when I get ready to start reviewing on Monday afternoon. I'm not opposed to like a week? Unless that's longer than what's possible.